cmd/rocsp-tool/main.go

```diff
@@ -495,6 +509,15 @@ func (cl *client) signAndStoreResponses(ctx context.Context, input <-chan *sa.Ce
 	}
 }
 
+func (cl *client) findIssuerShortID(longID int64) (byte, error) {
```

Comment:
I'd recommend having this function and `findIssuer` have the same return type `(ShortIDIssuer, error)`, defining them at the same place in the file (right after the ShortIDIssuer struct), and having their only difference be their input: are we doing lookup via a certStatus row (i.e. an Issuer(Name)ID int64) or are we doing lookup via an existing OCSP Response? Then they can be named `findIssuerByID` and `findIssuerByName` or something like that.